configure.py

```diff
@@ -1114,6 +1122,30 @@ def isa_flags_for(self, isa, arch):
             return self.isa_flags[arch_isa]
         return None
 
+    def get_isa_specific_flags(self, isas, arch):
+        flags = set()
+
+        def simd32_impl():
+            for simd_isa in ['sse2', 'altivec', 'neon']:
+                if simd_isa in arch.isa_extensions and self.isa_flags_for(simd_isa, arch.basename):
+                    return simd_isa
+            return None
+
+        for isa in isas:
+
+            if isa == 'simd':
+                isa = simd32_impl()
+
+                if isa is None:
+                    continue
+
+            flagset = self.isa_flags_for(isa, arch.basename)
+            if flagset is None:
+                raise UserError('Compiler %s does not support %s' % (self.basename, isa))
+            flags.add(flagset)
+
+        return ("".join([" %s" % f for f in sorted(flags)])).strip()
```

Comment:
Isn't this the same as `" ".join(sorted(flags))`?

Reply:
I believe so, will change. Thanks